src/components/Form/stories/Form.check.group.stories.mdx

```diff
@@ -32,8 +32,8 @@ For the non-textual checkbox and radio controls, FormCheck provides a single com
     <Form>
       <Form.Group>
         <Form.CheckGroup label="Sample question">
-          <Form.Check type="radio" label={`checkbox 1`} />
-          <Form.Check type="radio" label={`checkbox 2`} />
+          <Form.Check type="radio" label={`checkbox 1`} name="sameName" />
+          <Form.Check type="radio" label={`checkbox 2`} name="sameName" />
```

Comment:
Can you use a better name? Something like Group name or something more specific? It confused me the first time

Reply:
changed it